docs/en/operations/clickhouse-keeper.md

```diff
@@ -106,13 +106,14 @@ Seamlessly migration from ZooKeeper to `clickhouse-keeper` is impossible you hav
 
 1. Stop all ZooKeeper nodes.
 
-2. [optional, but recommended] Found ZooKeeper leader node, start and stop it again. It will force ZooKeeper to create consistent snapshot.
+2. Optional, but recommended: find ZooKeeper leader node, start and stop it again. It will force ZooKeeper to create a consistent snapshot.
 
-3. Run `clickhouse-keeper-converter` on leader, example
+3. Run `clickhouse-keeper-converter` on a leader, for example:
 
 ```bash
 clickhouse-keeper-converter --zookeeper-logs-dir /var/lib/zookeeper/version-2 --zookeeper-snapshots-dir /var/lib/zookeeper/version-2 --output-dir /path/to/clickhouse/keeper/snapshots
 ```
 
-4. Copy snapshot to `clickhouse-server` nodes with configured `keeper` or start `clickhouse-keeper` instead of ZooKeeper. Snapshot must persist on all nodes, otherwise empty nodes can be faster and one of them can becamse leader.
+4. Copy snapshot to ClickHouse server nodes with configured `keeper` or start `clickhouse-keeper` instead of ZooKeeper. The snapshot must persist on all nodes, otherwise, empty nodes can be faster and one of them can become leader.
```

Comment:
```suggestion
4. Copy snapshot to ClickHouse server nodes with configured `keeper` or start `clickhouse-keeper` instead of ZooKeeper. The snapshot must persist on all nodes, otherwise, empty nodes can be faster and one of them can become a leader.
```